src/vdbfusion/pybind/vdbfusion_pybind.cpp

```diff
@@ -51,6 +53,16 @@ using namespace py::literals;
 
 namespace vdbfusion {
 
+py::array_t<float> ToNumpy(openvdb::FloatGrid::Ptr grid) {
```

Comment:
Very nice utility, have you checked also [vdb_to_numpy](https://github.com/PRBonn/vdb_to_numpy) ? There I have some utils to extract just the leaf nodes, avoiding gignatic volumes on the RAM

Reply:
I have had a quick look, but I like to have to option to directly access the arrays without storing them to disk first and I never really got the `openvdb` integration working.

Reply:
By the way I changed this function to also return the minium and maximum coordinates of the grid, so it is possible to reconstruct the griding with `np.linespace(mins[0], maxs[0], tsdf.shape[0])`, etc.